tests/shakedown/shakedown/dcos/service.py

```diff
@@ -387,21 +388,8 @@ def service_unavailable_predicate(service_name):
     return response.status_code == 500
 
 
-def wait_for_service_endpoint(service_name, timeout_sec=120):
```

Comment:
We have a different version in `tests/system/common.py`. I'll import them once #6638 and #6642 landed.